exercises/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,27 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number1) {
+  let result = number1 < 0;
+  return result;
+}
+
+function isBetween5and10(number1) {
+  let result = number1 <= 10 && number1 > 5;
```

Comment:
I think this function should only be true for 6, 7, 8, 9. 
But right now it will return true for 10.